### PR TITLE
Routes refactor 

### PR DESCRIPTION
**Pull request by lssotog**
## Summary
_Refactor routes including loadComponent_

## Details
_It was included laodComponent to the app.routes.ts files._

### References

image

### Checks

- [ ] Unit test 
- [ ] a11y 
- [ ] Semantic

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-import { HomeComponent } from './home/home.component';
-import { LoginComponent } from './login/login.component';
-import { ProfileComponent } from './profile/profile.component';
-import { RegisterComponent } from './register/register.component';
-import { NavBarComponent } from './components/nav-bar/nav-bar.component';
+import { HomeComponent } from './pages/home/home.component';
+import { LoginComponent } from './pages/login/login.component';
+import { ProfileComponent } from './pages/profile/profile.component';
+import { RegisterComponent } from './pages/register/register.component';
+import { NavBarComponent } from './shared/components/molecules/nav-bar/nav-bar.component';
```
Comment (harleysg) [inline src/app/app.component.ts:7]: No es necesario importar los componentes. Por eso vez que no te funcionan el lazy-loading. te aparecen en el source del browser por que aqui se cargan.

File: src/app/pages/register/register.component.ts [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import { Component } from '@angular/core';
+import { SharedModule } from '../../shared/common/shared.module';
+import { RouterLink } from '@angular/router';
```
Comment (harleysg) [inline src/app/pages/register/register.component.ts:3]:
```suggestion
import { RouterLink } from '@angular/router';

import { SharedModule } from '../../shared/common/shared.module';
```

File: src/app/shared/common/shared.module.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    ButtonComponent,
+    FooterComponent,
+    HeaderComponent,
+    InputComponent,
+    NavBarComponent
+  ],
+  providers: [],
+})
+
+export class SharedModule {}
```
Comment (harleysg) [inline src/app/shared/common/shared.module.ts:29]:
```suggestion
export class SharedModule {}

```

File: src/app/shared/common/shared.module.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { NgModule } from "@angular/core";
+import { AuthorSignComponent } from "../components/atoms/author-sign/author-sign.component";
```
Comment (harleysg) [inline src/app/shared/common/shared.module.ts:2]:
```suggestion

import { AuthorSignComponent } from "../components/atoms/author-sign/author-sign.component";
```

File: src/app/app.routes.ts [hunk 5]
```diff
@@ -1,14 +1,14 @@
 import { Routes } from '@angular/router';
-import { HomeComponent } from './home/home.component';
-import { LoginComponent } from './login/login.component';
-import { ProfileComponent } from './profile/profile.component';
-import { RegisterComponent } from './register/register.component';
+// import { HomeComponent } from './pages/home/home.component';
```
Comment (harleysg) [inline src/app/app.routes.ts:2]:
```suggestion
```

File: src/app/pages/login/login.component.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.scss'
+})
+export class LoginComponent {
+  faEnvelope = faEnvelope;
+  faGoogle = faG;
+  copyText = copyText;
+  // faFacebook = faFacebook;
+  // faTwitter = faTwitter;
+  // faGithub = faGithub;
```
Comment (harleysg) [inline src/app/pages/login/login.component.ts:23]:
```suggestion
```

File: src/app/pages/login/login.component.ts [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { faG,  faEnvelope } from '@fortawesome/free-solid-svg-icons';
```
Comment (harleysg) [inline src/app/pages/login/login.component.ts:3]:
```suggestion

import { faG,  faEnvelope } from '@fortawesome/free-solid-svg-icons';
```

File: src/app/app.component.ts [hunk 8]
```diff
@@ -1,10 +1,10 @@
 import { Component } from '@angular/core';
 import { RouterOutlet } from '@angular/router';
-import { HomeComponent } from './home/home.component';
-import { LoginComponent } from './login/login.component';
-import { ProfileComponent } from './profile/profile.component';
-import { RegisterComponent } from './register/register.component';
-import { NavBarComponent } from './components/nav-bar/nav-bar.component';
+import { HomeComponent } from './pages/home/home.component';
```
Comment (harleysg) [inline src/app/app.component.ts:3]:
```suggestion

import { HomeComponent } from './pages/home/home.component';
```

**Review by harleysg — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (lssotog) [reply src/app/app.component.ts:7]: I deleted the components except the NavBarComponent bc it's been called on the html 

image

File: src/app/pages/login/login.component.ts (hunk 6 above)
Comment (lssotog) [reply src/app/pages/login/login.component.ts:23]: Done. 
image

File: src/app/app.routes.ts (hunk 5 above)
Comment (lssotog) [reply src/app/app.routes.ts:2]: Done
image

File: src/app/pages/register/register.component.ts (hunk 2 above)
Comment (lssotog) [reply src/app/pages/register/register.component.ts:3]: Done
image

**Review by lssotog — COMMENTED** [review]
Suggestions accepted

**Pull request merged**